File: CPP/Aksesoris.cpp
#include "PetShop.cpp"

class Aksesoris : public PetShop{
private:
    string jenis;
    string bahan;
    string warna;

public:
    // Constructor
    Aksesoris() : PetShop(){ 
    }

    Aksesoris(int id, string nama, double harga, int stok, string jenis, string bahan, string warna) : PetShop(id, nama, harga, stok){
        this->jenis = jenis;
        this->bahan = bahan;
        this->warna = warna;
    }

    // Setter dan Getter

    // Setter jenis
    void setJenis(string jenis){
        this->jenis = jenis;
    }

    // Setter bahan
    void setBahan(string bahan){
        this->bahan = bahan;
    }

    // Setter warna
    void setWarna(string warna){
        this->warna = warna;
    }

    // Getter jenis
    string getJenis(){
        return jenis;
    }

    // Getter bahan
    string getBahan(){
        return bahan;
    }

    // Getter warna
    string getWarna(){
        return warna;
    }

    ~Aksesoris(){
    }
};

File: PHP/Aksesoris.php
<?php
include_once 'PetShop.php';

class Aksesoris extends PetShop {
    private $jenis;
    private $bahan;
    private $warna;

    public function __construct($id = null, $nama = null, $harga = null, $stok = null, $foto_produk = null, $jenis = null, $bahan = null, $warna = null) {
        parent::__construct($id, $nama, $harga, $stok, $foto_produk);
        $this->jenis = $jenis;
        $this->bahan = $bahan;
        $this->warna = $warna;
    }

    // setter dan getter

    // setter jenis
    public function setJenis($jenis) {
        $this->jenis = $jenis;
    }

    // setter bahan
    public function setBahan($bahan) {
        $this->bahan = $bahan;
    }

    // setter warna
    public function setWarna($warna) {
        $this->warna = $warna;
    }

    // getter jenis
    public function getJenis() {
        return $this->jenis;
    }

    // getter bahan
    public function getBahan() {
        return $this->bahan;
    }

    // getter warna
    public function getWarna() {
        return $this->warna;
    }
}
?>